tests/test-utils.go

```diff
@@ -0,0 +1,21 @@
+package tests
```

Comment:
You may want to call this `tests/mock-attr.go`, instead of the generic name of `test-utils.go`